src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -664,4 +664,9 @@ inline std::istream& operator>>(std::istream& is, Affinity& affinity) {
  * environment variable is set (as affinity is configured explicitly)
  */
 static constexpr Property<Affinity> affinity{"AFFINITY"};
+
+/**
+ * @brief The name for setting auto-batching option.
+ */
+static constexpr Property<bool, PropertyMutability::RW> allow_auto_batching{"ALLOW_AUTO_BATCHING"};
```

Comment:
here it's also introduced as a hint https://github.com/openvinotoolkit/openvino/pull/10149

Reply:
yes, pls rebase upon #10149 merge

Reply:
@myshevts @ilya-lavrenov   updated the PR upon #10149 , please help review again